digit-health-services/health-delivery-service/.gitignore

```diff
@@ -0,0 +1,5 @@
+target/
+egov-pg-service.iml
+*.iml
+health-sync-service.iml
+src/main/resources/config/
```

Comment:
should we update the gitignore file for the new service?

Reply:
updated